src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="product">
+      <div class="product__card" data-qa="card">
+
+        <img class="picture" src="../src/images/imac.jpeg"
+        alt="image">
+
+        <h2 class="product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <span class="product__code">Product code: 195434</span>
+
+        <div class="product__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star active"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="product__price">
+
+          <span class="price">Price:</span>
+
+          <span class="sum">$2,199</span>
```

Comment:
due to BEM it should be 'product__price-label' and 'product__price-amount' for example